vue-frontend/src/components/portfolio/FooterSection.vue

```diff
@@ -12,9 +12,133 @@
       />
     </aspect-ratio>
   </section>
-
   <section
-    class="v2-header-2-text tw-font-bold tw-text-center tw-bg-white tw-py-20 sm:tw-py-40 lg:tw-py-80"
+    class="tw-bg-white tw-pt-36 md:tw-pb-0 lg:tw-pt-50"
+    :ref="response.review ? response.review.ref : ''"
+  >
+    <div class="header-text canopas-gradient-text tw-text-center">
+      Listen what real app users <br />
+      are saying...
+    </div>
+    <div
+      class="tw-mt-[32px] tw-text-left tw-overflow-x-hidden tw-overflow-y-auto tw-scrollbar-0"
+      ref="horizontalGrid"
+    >
+      <div
+        class="tw-animate-gridAnimationReverse hover:tw-animate-gridAnimationReverse"
+        :class="gridAnimationReverse"
+      >
+        <div
+          class="tw-animate-gridAnimation hover:tw-animate-gridAnimationPaused"
+        >
+          <div
+            v-if="usersReviewsGrid1"
+            class="tw-flex tw-mb-[16px] tw-ml-[160px] sm:tw-ml-[240px] lg:tw-ml-[264px] tw-w-full"
+            ref="horizontalGrid1"
+          >
+            <div
+              v-for="userReview in usersReviewsGrid1"
```

Comment:
You can use for loop for appending grid 3 times, instead of adding them manually